auth-service/src/lib/utils.ts

```diff
@@ -31,3 +32,21 @@ export function delay(ms: number) {
         setTimeout(() => resolve(), ms);
     });
 }
+
+/**
+ * Return item from redis wrapped in promise
+ * 
+ * @param key key of the item
+ * @param redis RedisClient
+ */
+export function getFromRedis(key: string, redis: RedisClient): Promise<any> {
```

Comment:
This is quite nice. I'm not sure how many Redis APIs we're gonna use, but in case it grows at some point, maybe we could implement a wrapper class or use `util.promisify`.

Reply:
Yeh if theres more functionality needed wrapper class would come handy.